mlflow/java/client/src/test/java/org/mlflow/tracking/MlflowProtobufMapperTest.java

```diff
@@ -0,0 +1,39 @@
+package org.mlflow.tracking;
+
+import java.lang.reflect.Type;
+import java.util.List;
+import java.util.Map;
+
+import com.beust.jcommander.internal.Maps;
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import org.mlflow.api.proto.Service;
+
+public class MlflowProtobufMapperTest {
+  @Test
+  public void testSerializeSnakeCase() {
+    MlflowProtobufMapper mapper = new MlflowProtobufMapper();
+    String result = mapper.makeLogParam("my-id", "my-key", "my-value");
+
+    Gson gson = new Gson();
+    Type type = new TypeToken<Map<String, Object>>(){}.getType();
+    Map<String, String> serializedMessage = gson.fromJson(result, type);
+
+    Map<String, String> expectedMessage = Maps.newHashMap();
```

Comment:
Very nit: `Maps.newHashMap` seems to be the equivalent of `new HashMap<>()`. If we want value from the jcommander Maps module in this case, it might make sense to use the constructor defined here: https://github.com/cbeust/jcommander/blob/9f251415677106b683e4bbaf82123f0e5930cdce/src/main/java/com/beust/jcommander/internal/Maps.java#L35.